WhoIsThat/WhoIsThat/HomePage.xaml

```diff
@@ -1,12 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             x:Class="WhoIsThat.HomePage">
+             x:Class="WhoIsThat.HomePage"
+             Padding="60, 5, 60, 30">
     <ContentPage.Content>
         <StackLayout>
-            <Label Text="Welcome to Xamarin.Forms!"
-                VerticalOptions="CenterAndExpand" 
-                HorizontalOptions="CenterAndExpand" />
+            <Button Text="Take a photo!"
```

Comment:
You should put such strings into `Strings.xml` to be able to localize it

Reply:
As I tried to use shared code, did not find a way to reference strings as resources (as in android I should just go with "@string/my_string").